src/Compilers/CSharp/Test/Semantic/Semantics/WarningVersionTests.cs

```diff
@@ -139,13 +139,67 @@ public Program(int dummy)
     {
     }
 }";
-            CreateCompilation(source2, references: new MetadataReference[] { moduleReference }, options: TestOptions.ReleaseDll.WithWarningLevel(CodeAnalysis.Diagnostic.DefaultWarningLevel)).VerifyDiagnostics(
-                );
-            CreateCompilation(source2, references: new MetadataReference[] { moduleReference }, options: TestOptions.ReleaseDll.WithWarningLevel(5)).VerifyDiagnostics(
-                // (4,12): warning CS8822: Auto-implemented property 'Program.Property' must be fully assigned before control is returned to the caller.
+            var csharp10IL = @"
+{
+  // Code size        2 (0x2)
+  .maxstack  0
+  IL_0000:  nop
+  IL_0001:  ret
+}
+";
+            var csharp11IL = @"
+{
+  // Code size       14 (0xe)
+  .maxstack  1
+  IL_0000:  ldarg.0
+  IL_0001:  ldflda     ""Struct Program.<Property>k__BackingField""
+  IL_0006:  initobj    ""Struct""
+  IL_000c:  nop
+  IL_000d:  ret
+}
+";
+            // C# 10
+            // note: no LangVersion diagnostics are given here, even though we will change codegen in the later language version
+            var verifier = CompileAndVerify(
+                source2,
+                references: moduleReference,
+                options: TestOptions.DebugDll.WithWarningLevel(CodeAnalysis.Diagnostic.DefaultWarningLevel),
+                parseOptions: TestOptions.Regular10,
+                verify: Verification.Skipped);
```

Comment:
>Verification.Skipped [](http://example.com/codeflow?start=24&length=20)

Why skip verification in these cases?

Reply:
I think it's related to the use of a netmodule here. I get the following error when I remove the 'verify' argument (where the GUID is different each run):

```cs
System.Exception : IL Verify failed unexpectedly: 
ILVerify was not able to resolve a module named 'fa651942-7f13-469b-aa0f-14134e5a9532'
```